recipes/simfil/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+import os
+
+
+required_conan_version = ">=1.62.0"
```

Comment:
```suggestion
required_conan_version = ">=1.53.0"
```

This recipe does not require 1.6x.x specific features.

Reply:
How do I know?

Reply:
There is no easy way, need to read the changelog and compare if you need some feature from there: https://docs.conan.io/1/changelog.html

also, most of tools has which version they are available, e.g.

https://docs.conan.io/1/reference/conanfile/tools/files/basic.html#conan-tools-files-save